SessionUtilitiesKit/Utilities/BencodeResponse.swift

```diff
@@ -15,13 +15,16 @@ extension BencodeResponse: Decodable {
         info = try {
             /// First try to decode it directly
             if let info: T = try? container.decode(T.self) {
+                SNLog("Successfully decoded info directly")
                 return info
             }
             
             /// If that failed then we need to reset the container and try decode it as a JSON string
             container = try decoder.unkeyedContainer()
             let infoString: String = try container.decode(String.self)
+            SNLog("Successfully decoded info to JSON string")
             let infoData: Data = try infoString.data(using: .ascii) ?? { throw NetworkError.parsingFailed }()
+            SNLog("Successfully decoded info to JSON data")
```

Comment:
Might be good to remove these logs now (not sure we would get much benefit from them in users logs in prod 🤔 )